### DISCUSSION TITLE
variable affectation from a script call

### DISCUSSION THREAD

**Discussion by sisyphe999** [Pavel-Husakouski/x-var#2] [Q&A] [answered]
Hi
Can I use x-var to perform this kind of script call?
"ip": "x-var MYIP=$(curl -s http://ipinfo.io/ip) && x-var echo $MYIP",

**Comment by Pavel-Husakouski** [comment, marked as answer]
`x-var MYIP=$(curl -s http://ipinfo.io/ip) echo \$MYIP`
